pkg/catalog/ingress.go

```diff
@@ -8,12 +8,13 @@ import (
 	"github.com/openservicemesh/osm/pkg/trafficpolicy"
 )
 
+// Ingress does not depend on k8s service accounts, program a wildcard (empty struct) to indicate
+// to RDS that an inbound traffic policy for ingress should not enforce service account based RBAC policies.
+var wildcardServiceAccount = service.K8sServiceAccount{}
```

Comment:
🤔 I'll admit that I find this `wildcardServiceAccount` and a blank `ServiceAccount` a bit confusing - the comment helps.

Reply:
Since the RDS logic to build routes is the same for ingress and in-mesh policies, but ingress is not associated with a service account, this is used to to program a principal of `ANY` in the RBAC rule for this route. ANY means allow all downstreams, not just a downstream that matches a specific service account principal in an RBAC policy.